src/clusterfuzz/_internal/issue_management/issue_tracker_utils.py

```diff
@@ -43,8 +44,16 @@ def _get_issue_tracker_project_name(testcase=None):
   return data_handler.get_issue_tracker_name(job_type)
 
 
-@memoize.wrap(memoize.FifoInMemory(256))
-def get_issue_tracker(project_name=None):
+def get_issue_tracker(project_name=None, is_appengine=True):
+  """Get the issue tracker with the given type and name."""
+  if is_appengine:
+    return request_cache.wrap(capacity=_ISSUE_TRACKER_CACHE_CAPACITY)(
```

Comment:
We should move request_cache back to appengine since it's very specific to appengine. We can jsut defer the import here..